bat-utils/lib/youtube.test.js

```diff
@@ -1,6 +1,8 @@
 'use strict'
 import test from 'ava'
 import getYoutubeChannelId from './youtube'
+import dotenv from 'dotenv'
```

Comment:
why do we need this?

Reply:
i found that i have to source the env to prevent the test from failing unless this line is in. if i recall correctly, ava starts a new process, so if it's not ones' env already (with `npm t`) test fails